packages/yew/src/html/conversion.rs

```diff
@@ -309,13 +315,13 @@ mod test {
             html! {
                 <div>
                     <header>
-                        {header}
+                        {for header}
```

Comment:
This should be the thing that is not possible: Render `ChildrenRenderer<VChild<T>>` into a `VNode`.

This was previously possible because the compiler can reliably infer `ChildrenRenderer<_>` by specifying the type at `children: ChildrenRenderer<_>`. This not possible anymore because it now converts to `VNode`.

```rust
impl IntoPropValue<VNode> for ChildrenRenderer<VChild<T>> {}
impl IntoPropValue<VNode> for ChildrenRenderer<VNode> {}
```


Having both implementations will confuse the rust compiler. to infer what `_` is in `ChildrenRenderer<_>`.
So I picked `Children`, which should be way more common.

Reply:
Wouldn't `VNode` be also fine for this? `{vnode}` should work fine

Reply:
`header` has type of `ChildrenRenderer<VChild<T>>`, specified in the properties.
If `header` is of type `VNode`, then `{vnode}` would work.